bsi_zoo/tests/test_estimators.py

```diff
@@ -15,7 +15,6 @@
 )
 
 
-# TODO: make the iterative type-II methods more efficient
 @pytest.mark.parametrize("n_times", [5])
 # @pytest.mark.parametrize("orientation_type", ["fixed", "free"])
 @pytest.mark.parametrize("orientation_type", ["free"])
```

Comment:
green @AliHashemi-ai !

now can we put back the fixed in the test too?

Reply:
Sure @agramfort Let's do this. Shall I do it in this branch?

Reply:
yes try and if it's more complex than expected we'll do another PR.

Reply:
> yes try and if it's more complex than expected we'll do another PR.

I change the code here https://github.com/braindatalab/BSI-Zoo/blob/n_orient_3/bsi_zoo/estimators.py#L61 and now all the tests for fixed and free orientation are passed